gnovm/pkg/gnolang/utils.go

```diff
@@ -19,3 +25,27 @@ func endsWith(item string, suffixes []string) bool {
 	}
 	return false
 }
+
+// TODO(hariom): move to better place
+func ParseMemMod(dir string) *std.MemMod {
+	var memMod std.MemMod
```

Comment:
Why declare here, when you're actually setting the object at the end of the method?